test/utilities/navigationUtilities.spec.js

```diff
@@ -10,3 +17,26 @@ ava('assembleProceduralRouteString() concatenates provided strings with appropri
 
     t.true(result === expectedResult);
 });
+
+ava('getCircularCoordinates() provides a set of points equidistant from center', (t) => {
+    const TOLERABLE_ERROR = 0.025  // 25m
+
+    const posAndDecl = [airportModelFixture.positionModel, MAGNETIC_NORTH_MOCK];
+    const center = GPS_COORDINATES_MOCK;
+    const radius = 1.0;
+
+    const circle = getCircularCoordinates(center, radius, posAndDecl);
+    t.true(circle.length === 32);
+
+    const trueCenter = DynamicPositionModel.calculateRelativePosition(center, ...posAndDecl)
+
+    let ok = true;
```

Comment:
you can simplify this entire block using `t.plan()` (https://github.com/avajs/ava/blob/master/docs/03-assertions.md#assertion-planning):

```js
t.plan(circle.length);

for (let i = 0; i < circle.length; i++) {
    const distance = distance2d(trueCenter, circle[i]);
    const isWithinTolerance = abs(distance - km(1)) <= TOLERABLE_ERROR;

    t.true(isWithinTolerance);
}
```